examples/advance_music_cog.py

```diff
@@ -0,0 +1,457 @@
+            else:
+                loop_status = "Looping Disabled"
+            
+            #fecthing other details   
+            thumbnail = player.data['videoDetails']['thumbnail']['thumbnails'][-1]['url']
+            title = player.data['videoDetails']['title']
+            url = player.url
+            uploader = player.data['videoDetails']['author']
+            views = player.data['videoDetails']['viewCount']
+            rating = player.data['videoDetails']['averageRating']
+            
+            #creating embed
+            embed = discord.Embed(
+                title = 'Now playing',
+                description = f"**{title}**",
+                timestamp = datetime.datetime.utcnow(),
+                color = discord.Color.from_rgb(0, 255, 255)
+                )
+            embed.add_field(
+                name = 'Played',
+                value = parse_duration(duration_played)
+                )
+            embed.add_field(
+                name = 'Duration',
+                value = parse_duration(player.duration)
+                )
+            embed.add_field(
+                name = 'Loop',
+                value = loop_status
+                )
+            embed.add_field(
+                name = 'Requested by',
+                value = player.requester.mention
+                )
+            embed.add_field(
+                name = 'Uploader',
+                value = uploader
+                )
+            embed.add_field(
+                name = 'URL',
+                value = f'[Click]({url})'
+                )
+            embed.add_field(
+                name = 'Views',
+                value = parse_count(int(views))
+                )
+            embed.add_field(
+                name ='Rating',
+                value = rating
+                )
+            embed.set_thumbnail(
+                url = thumbnail
+                )
+            embed.set_image(
+                url = r"https://i.imgur.com/ufxvZ0j.gif"
+                )
+            embed.set_author(
+                name = ctx.author.name,
+                icon_url = ctx.author.avatar_url
+                )
+
+            #sending the embed
+            await ctx.send(embed=embed)
+
+    #join command        
+    @commands.command()
+    async def join(self, ctx):
+        if await self.MusicManager.join(ctx):
+            await ctx.send("Joined Voice Channel")
+
+    #play command
+    @commands.command()
+    async def play(self, ctx, *, query: str):
+        #checking if the bot has joined a voice channel
+        if not ctx.voice_client or not ctx.voice_client.is_connected():
+            await self.MusicManager.join(ctx)
+
+        #searching while showing typing status
+        async with ctx.typing(): 
+            players = await self.MusicManager.create_player(query, ctx.author)
+        #^change to async ctx.defer() if using slash commands
+
+        #if a player object is created ie. song found
```

Comment:
For example here, instead of 
```py
#if a player object is created ie. song found
```
do:
```py
# If a player object is create ie. song found
```

Reply:
Make sure this is used in ALL comments